### DISCUSSION TITLE
Sudden Death and Perfect Mod needs a proper failscreen

### DISCUSSION THREAD

**Discussion by NotGumballer91** [ppy/osu#14146] [Ideas] [answered]
I would like to see specifically which circle/slider I missed. Instantly restarting it seems a little weird to me. Like retry-spamming without pressing anything. 

I personally think it's better to have normal failedscreen (Retry and Quit) for both SD and PF. And you can just hold ~ to quick-restart anyway. It feels more intuitive to me.
I would rather see this than instant-restart. Gives a little breathing room for the player to see what happened.
![osu_2021-08-06_19-48-19](https://user-images.githubusercontent.com/64581009/128506130-7f6bac31-380c-4e7c-8bc3-d1c78d5371a5.png)
_Oh maybe make the missed circle/slider in red tint?_ That would be cool

**Comment by hlysine** [comment, marked as answer]
This is in the works in #11200. It lets you choose whether to see this failed screen for SD PF or not.
